simulator-gradle-plugin/src/main/kotlin/org/robolectric/simulator/gradle/SimulatorPlugin.kt

```diff
@@ -31,7 +36,7 @@ class SimulatorPlugin : Plugin<Project> {
 
     val testTaskName = "testDebugUnitTest"
     val testTask = project.tasks.findByName(testTaskName) as? Test
-    val buildDir = project.buildDir
+    val buildDir = project.layout.buildDirectory.asFile.get()
```

Comment:
I would use the following here:
```suggestion
    val buildDir = project.layout.buildDirectory.get()
```

And use this for `resourceApkFile` below:
```kotlin
val resourceApkFile =
  buildDir.file("intermediates/apk_for_local_test/debugUnitTest/apk-for-local-test.ap_").asFile
```